src/main/java/blackjack/domain/CardManager.java

```diff
@@ -0,0 +1,45 @@
+package blackjack.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class CardManager {
+
+    private final CardDeck cardDeck;
+    private final Map<Nickname, Cards> nicknameToCards = new HashMap<>();
+
+    public CardManager() {
+        cardDeck = CardDeck.initialize();
+    }
+
+    public void initialize(List<Nickname> nicknames) {
+        nicknames.forEach(
+                nickname -> nicknameToCards.put(nickname, Cards.initialize())
+        );
+    }
+
+    public void distributeCards() {
+        Set<Nickname> nicknames = nicknameToCards.keySet();
+        nicknames.forEach(nickname ->
+                nicknameToCards.get(nickname)
+                        .add(cardDeck.drawCard(GameRule.INITIAL_CARD_COUNT.getValue()))
+        );
```

Comment:
매번 `Set<Nickname> nicknames = nicknameToCards.keySet();`로 선언하지 않고 
`nicknameToCards.values()`를 활용할 수 있을 것 같아요.

```suggestion
    nicknameToCards.values().forEach(cards ->
            cards.add(cardDeck.drawCard(GameRule.INITIAL_CARD_COUNT.getValue()))
    );
```

Reply:
너무 좋네요. 완전히 옳습니다!